lib/modules/manager/devbox/artifacts.ts

```diff
@@ -0,0 +1,67 @@
+import { logger } from '../../../logger';
+import { exec } from '../../../util/exec';
+import type { ExecOptions } from '../../../util/exec/types';
+import { readLocalFile } from '../../../util/fs';
+import { getRepoStatus } from '../../../util/git';
+import type { UpdateArtifact, UpdateArtifactsResult } from '../types';
+
+export async function updateArtifacts(
+  updateConfig: UpdateArtifact,
+): Promise<UpdateArtifactsResult[] | null> {
+  const lockFileName = updateConfig.packageFileName.replace(
+    /devbox.json$/,
+    'devbox.lock',
+  );
+  const existingLockFileContent = await readLocalFile(lockFileName, 'utf8');
+  if (!existingLockFileContent) {
+    logger.debug('No devbox.lock found');
+    return null;
+  }
+  const execOptions: ExecOptions = {
+    cwdFile: updateConfig.packageFileName,
+    toolConstraints: [
+      {
+        toolName: 'devbox',
+        constraint: updateConfig.config.constraints?.devbox,
+      },
+    ],
+    docker: {},
+    userConfiguredEnv: updateConfig.config.env,
+  };
+
+  let cmd = '';
+  if (updateConfig.config.isLockFileMaintenance) {
```

Comment:
You answered supportsLockFileMaintenance=false in the index file, which contradicts this

Reply:
I will update this